packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -1,329 +1,364 @@
+        shippingTitle,
+        submitOrder,
+        submitting
+    } = props;
+
+    const handleDismiss = useCallback(() => {
+        cancelCheckout();
+    }, [cancelCheckout]);
+
+    const handleAddressFormClick = useCallback(() => {
+        editOrder('address');
+    }, [editOrder]);
+
+    const handlePaymentFormClick = useCallback(() => {
+        editOrder('paymentMethod');
+    }, [editOrder]);
+
+    const handleShippingFormClick = useCallback(() => {
+        editOrder('shippingMethod');
+    }, [editOrder]);
+
+    return (
+        <Fragment>
+            <div className={classes.body}>
+                <Section
+                    label="Ship To"
+                    onClick={handleAddressFormClick}
+                    showEditIcon={hasShippingAddress}
+                >
+                    <ShippingAddressSummary
+                        classes={classes}
+                        hasShippingAddress={hasShippingAddress}
+                        shippingAddress={shippingAddress}
+                    />
+                </Section>
+                <Section
+                    label="Pay With"
+                    onClick={handlePaymentFormClick}
+                    showEditIcon={hasPaymentMethod}
+                >
+                    <PaymentMethodSummary
+                        classes={classes}
+                        hasPaymentMethod={hasPaymentMethod}
+                        paymentData={paymentData}
+                    />
+                </Section>
+                <Section
+                    label="Use"
+                    onClick={handleShippingFormClick}
+                    showEditIcon={hasShippingMethod}
+                >
+                    <ShippingMethodSummary
+                        classes={classes}
+                        hasShippingMethod={hasShippingMethod}
+                        shippingTitle={shippingTitle}
+                    />
+                </Section>
+                <Section label="TOTAL">
+                    <Price
+                        currencyCode={cart.totals.quote_currency_code}
+                        value={cart.totals.subtotal || 0}
+                    />
+                    <br />
+                    <span>{cart.details.items_qty} Items</span>
+                </Section>
+            </div>
+            <div className={classes.footer}>
+                <Button onClick={handleDismiss}>Back to Cart</Button>
+                <Button
+                    priority="high"
+                    disabled={isSubmitDisabled(submitting, ready)}
+                    onClick={submitOrder}
+                >
+                    Confirm Order
+                </Button>
+            </div>
+        </Fragment>
+    );
+};
+
+/**
+ * The Form component is similar to Flow in that it renders either the overview
+ * or the editable form based on the 'editing' state value.
+ */
+const Form = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+    let child;
```

Comment:
Not that it's *super* important, but you should be able to use `const` even in places like this. We could tighten up the `prefer-const` rule if warnings would help.

```es6
const child = editing ? (
  <Foo />
) : (
  <Bar />
)

```